src/System.Runtime.CompilerServices.Unsafe/tests/UnsafeTests.cs

```diff
@@ -822,6 +822,116 @@ public static void Unbox_CustomValueType()
 
             Assert.Throws<InvalidCastException>(() => Unsafe.Unbox<bool>(box));
         }
+
+        [Fact]
+        public static void SkipInit()
+        {
+            // Validate that calling with primitive types works.
```

Comment:
I don't think there is a "good way" to validate that `Unsafe.SkipInit` leaves an "arbitrary" value, so I added these tests which just validate the call doesn't fail and the tests below (https://github.com/dotnet/corefx/pull/41995/commits/f2a593d680c1b1c541f157731b2f1a5d665305d3#diff-401abe3f2532c0a42128029a72a1240dR861) which validate that the previous value (w/e it was set to) is preserved.